5gms/5G_APIs-overrides/TS26512_CommonData.yaml

```diff
@@ -0,0 +1,244 @@
+openapi: 3.0.0
+info:
+  title: 5GMS Common Data Types
+  version: 2.0.1
+  description: |
+    5GMS Common Data Types
+    © 2023, 3GPP Organizational Partners (ARIB, ATIS, CCSA, ETSI, TSDSI, TTA, TTC).
+    All rights reserved.
+tags:
+  - name: 5GMS Common Data Types
+    description: '5G Media Streaming: Common Data Types'
+externalDocs:
+  description: 'TS 26.512 DRAFT V17.4.0; 5G Media Streaming (5GMS); Protocols'
+  url: 'https://www.3gpp.org/ftp/Specs/archive/26_series/26.512/'
+paths: {}
+components:
+  schemas:
+    #################################
+    # Clause 6.4.2: Simple data types
+    #################################
+    ResourceId:
+      type: string
+      description: String chosen by the 5GMS AF to serve as an identifier in a resource URI.
+    Percentage:
+      type: number
+      minimum: 0.0
+      maximum: 100.0
+    #DurationSec is defined in TS29571_CommonData
+    #DateTime is defined in TS29571_CommonData
+    #Uri is defined in TS29571_CommonData
+    Url:
+      type: string
```

Comment:
Could we tighten the syntax up by adding a `pattern` property with a regular expression?